lost/src/main/java/com/mapzen/android/lost/internal/FusedLocationProviderApiImpl.java

```diff
@@ -123,19 +125,25 @@ public boolean isConnected() {
 
   @Override public Location getLastLocation(LostApiClient client) {
     throwIfNotConnected(client);
+    Location location = null;
     try {
-      return service.getLastLocation();
+      location = service.getLastLocation();
     } catch (RemoteException e) {
-      throw new RuntimeException(e);
+      Log.e(TAG, "Error occurred trying to get last Location", e);
+    } finally {
+      return location;
```

Comment:
I would just add a note in the Javadoc for `getLastLocation()` that `null` is also a valid return value here and this looks good to go.